osu.Game/Screens/Play/PlayerSettings/InputSettings.cs

```diff
@@ -22,8 +23,9 @@ private void load(OsuConfigManager config)
             {
                 new PlayerCheckbox
                 {
-                    LabelText = MouseSettingsStrings.DisableClicksDuringGameplay,
-                    Current = config.GetBindable<bool>(OsuSetting.MouseDisableButtons)
+                    // TODO: change to touchscreen detection once https://github.com/ppy/osu/pull/25348 makes it in
+                    LabelText = RuntimeInfo.IsDesktop ? MouseSettingsStrings.DisableClicksDuringGameplay : TouchSettingsStrings.DisableTapsDuringGameplay,
+                    Current = config.GetBindable<bool>(RuntimeInfo.IsDesktop ? OsuSetting.MouseDisableButtons : OsuSetting.TouchDisableGameplayTaps)
```

Comment:
This change is a bit out of left field given the PR title and description but it's fine I guess.